tests/functional/contribute/test_friends.py

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.contribute.friends import ContributeFriendsPage
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_display_signup_form(base_url, selenium):
```

Comment:
Also - not a blocker for this PR, but it would be great to add tests for both a successful and failed form submission. To do this we would need to make sure the basket ID is configured correctly for all environments.